crates/goose-mcp/src/developer/developer_prompt.rs

```diff
@@ -0,0 +1,27 @@
+use include_dir::{include_dir, Dir};
+use mcp_core::prompt::{Prompt, PromptArgument, PromptTemplate};
+
+static PROMPTS_DIR: Dir = include_dir!("$CARGO_MANIFEST_DIR/src/developer/prompts");
+
+pub fn create_unit_test_prompt() -> Prompt {
+    let prompt_str = PROMPTS_DIR
```

Comment:
nit: probably worth it to iterate over the dir and load them all, for future contributors?

Reply:
nit: i'd call this file just prompt.rs since we're already in developer module